### ISSUE TITLE
Swift3, No such module 'ChameleonFramework'

### ISSUE THREAD

**Issue by zhugejunwei** [ViccAlexander/Chameleon#139]
I have installed ChameleonFramework through cocoaPods, but this error shows up on this line: 

`import ChameleonFramework`.

**Comment by zhugejunwei** [comment]
FYI, my Podfile:

platform :ios, '9.0'
use_frameworks!

target '8 Minute Workout' do

pod 'ChameleonFramework', '~> 2.1.0' 
pod 'MZTimerLabel', '~> 0.5.4' 
pod "youtube-ios-player-helper" 

end

**Comment by bre7** [comment]
The **Swift 3** branch hasn't been merged into master yet so try specifying the branch in your Podfile. Reopen if it pops up again though it doesn't seem to be an issue with Chameleon but rather your Podfile.

**Comment by zhugejunwei** [comment]
Thank you for your suggestion. But it still can't work. I specified the branch in my Podfile, and my Podfile is as bellow:

--------------------------------------------
platform :ios, '9.0'
use_frameworks!

target '8 Minute Workout' do

pod 'ChameleonFramework', :git => 'https://github.com/ViccAlexander/Chameleon.git', :branch => 'swift3'
pod 'MZTimerLabel', '~> 0.5.4' ## 计数器
pod "youtube-ios-player-helper" ## youtube视频

end
--------------------------------------------

Is this the right way to specify the branch? I did this just based on the official doc, http://guides.cocoapods.org/using/the-podfile.html.
